app.js

```diff
@@ -1,8 +1,43 @@
 // import functions and grab DOM elements
+import { signUpUser, signInUser, redirectToPolls } from './fetch-utils.js';
+//import { } from '../half-baked-soccer-keeper-feb/fetch-utils';
 
+const signUpForm = document.getElementById('sign-up');
+const signUpEmail = document.getElementById('sign-up-email');
+const signUpPassword = document.getElementById('sign-up-password');
+//const SignUpButton = document.querySelector('sign-up-button');
+console.log(signUpEmail);
+const signInForm = document.getElementById('sign-in');
+const signInEmail = document.getElementById('sign-in-email');
+const signInPassword = document.getElementById('sign-in-password');
+//const SignInButton = document.querySelector('sign-in-button');
 // let state
 
 // set event listeners 
+signUpForm.addEventListener('submit', async (e)=>{
+    e.preventDefault();
+    const user = await signUpUser(signUpEmail.value, signUpPassword.value);
+    console.log(`cookie?: ${user}`);
+    if (user){
+
+        redirectToPolls();
+    } else {
+        console.error(user);
+    }
+});
+
+signInForm.addEventListener('submit', async (e)=>{
+    e.preventDefault();
+    console.log(signInEmail.value, signInPassword.value);
```

Comment:
remember before a final push up of the code to go through and delete unnecessary console logs, it can be slightly disconcerting to a user to see their password on display in the console